site/e2e/google/protobuf/timestampGenerated.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable */
```

Comment:
I don't understand:
1. why this file needs to be generated given that we use `google.protobuf.Timestamp` elsewhere (I guess the provisioner is used in e2e somehow?)
2. WTF the linter issues are.

Please help

Reply:
1) `site/e2e` does use this.
2) You probably need to add it to `site/.eslintignore` and `site/.prettierignore` similar to what's already done for `provisionerGenerated.ts`

Reply:
1. Thanks!
2. Oh, I didn't add anything; this is all generated. I'l look into what's generating this.

Reply:
It's `gen:provisioner` in `site/package.json`. It looks like `protoc-gen-ts_proto` automatically generates it with that blanket `eslint-ignore` directive. I wouldn't worry about that and simply exclude it from being linted as it is a generated file.